addons/hw_drivers/iot_handlers/drivers/PrinterDriver_L.py

```diff
@@ -82,6 +82,7 @@ def __init__(self, identifier, device):
         self._actions.update({
             'cashbox': self.open_cashbox,
             'print_receipt': self.print_receipt,
+            'status': self.print_status,
```

Comment:
And about iot windows ???
PrinterDriver_W.py don't have this action
![image](https://github.com/user-attachments/assets/f90953f9-2512-4a07-abe7-4c8d265eeeb9)